packages/next/src/server/config.ts

```diff
@@ -767,6 +767,9 @@ function assignDefaults(
     '@mui/icons-material': {
       transform: '@mui/icons-material/{{member}}',
     },
+    '@phosphor-icons/react': {
+      transform: '@phosphor-icons/react/dist/ssr/{{member}}',
```

Comment:
Note: the `ssr` submodule is just a context-free version of the icons that can run on server or client. We assume isomorphic version here as the default, since use of `IconContext` is seemingly rare.